docs/src/tutorial/appendix.rst

```diff
@@ -28,4 +28,67 @@ procedure. Any contributions towards making the Windows install
 process smoother is welcomed; it is an unfortunate fact that none of
 the regular Cython developers have convenient access to Windows.
 
+Python 3.8+
+-----------
+
+Since Python 3.8, the search paths of DLL dependencies has been reset. ([changelog]_)
```

Comment:
```suggestion
Since Python 3.8, the search paths of DLL dependencies has been reset.
(`changelog <https://docs.python.org/3/whatsnew/3.8.html#changes-in-the-python-api>`_)
```